libs/orchestrator-ui-components/src/lib/components/SubscriptionsTabs/getSubscriptionsTabTypeFromString.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { getSubscriptionsTabTypeFromString } from './getSubscriptionsTabTypeFromString';
+import { SubscriptionsTabType } from './SubscriptionsTabs';
+
+describe('getSubscriptionsTabTypeFromString', () => {
+    it('returns ACTIVE', () => {
+        const result = getSubscriptionsTabTypeFromString(
+            SubscriptionsTabType.ACTIVE,
+        );
+        expect(result).toEqual(SubscriptionsTabType.ACTIVE);
+    });
+
+    it('returns TERMINATED', () => {
+        const result = getSubscriptionsTabTypeFromString(
+            SubscriptionsTabType.TERMINATED,
+        );
+        expect(result).toEqual(SubscriptionsTabType.TERMINATED);
+    });
+
+    it('returns TRANSIENT', () => {
+        const result = getSubscriptionsTabTypeFromString(
+            SubscriptionsTabType.TRANSIENT,
+        );
+        expect(result).toEqual(SubscriptionsTabType.TRANSIENT);
+    });
+
+    it('returns ALL', () => {
```

Comment:
also test normal string instead of only `SubscriptionsTabType`, `"all"` should also work right?

Reply:
You are right, the input must always be a string